### PR TITLE
Add githubstudent's pin to the map

### PR DESCRIPTION
**Pull request by blancopedro**
Adding Sevilla to the map!

### CODE REVIEW

**Comment by verde79** [comment]
@blancopedro brilliant!

File: _pins/githubstudents_json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: githubstudent
+latitude:37.389092
```
Comment (crichID) [inline _pins/githubstudents_json]: @blancopedro the reason the tests are failing is that you must have a space after the `:` on each line. If you click the Files changed tab, you will see a small pencil at the top right of the file. Click that and you will be back in edit mode! Please let me know if I can help!

**Comment by crichID** [comment]
@blancopedro you will also need to fix your file name. It is currently `_pins/githubstudents_json` but should be `_pins/githubstudents.json`

**Comment by crichID** [comment]
@blancopedro I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/3024/commits/7aa44ee9aa1e97761772fd8bae352d0cecf71ad8. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Pull request merged**